<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登陆</string>
  </property>
  <widget class="QLabel" name="loginwarningLabel">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>140</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="loginusrLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户：</string>
   </property>
  </widget>
  <widget class="QLabel" name="loginpasswordLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginclosePushButton">
   <property name="geometry">
    <rect>
     <x>443</x>
     <y>5</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginpassLineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>107</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="logincheckLabel">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>172</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>验证码：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginzhucePushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注  册</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="logincheckLineEdit">
   <property name="geometry">
    <rect>
     <x>303</x>
     <y>170</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loginforgetPushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>忘记密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginloginPushButton">
   <property name="geometry">
    <rect>
     <x>263</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>登  陆</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginexitPushbutton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退  出</string>
   </property>
  </widget>
  <widget class="QLabel" name="headLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>欢迎登陆</string>
   </property>
  </widget>
  <widget class="QLabel" name="logintimeLabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>287</y>
     <width>65</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="codeLayout"/>
  </widget>
  <widget class="QPushButton" name="loginbackgroundPushButton">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>4</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>修改背景</string>
   </property>
  </widget>
  <widget class="QListWidget" name="loginBackgroundListWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>时间煮雨</string>
    </property>
    <property name="icon">
     <iconset resource="background.qrc">
      <normaloff>:/background/1.jpg</normaloff>:/background/1.jpg</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>清新之风</string>
    </property>
    <property name="icon">
     <iconset resource="background.qrc">
      <normaloff>:/background/2.jpg</normaloff>:/background/2.jpg</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>淡化</string>
    </property>
    <property name="icon">
     <iconset resource="background.qrc">
      <normaloff>:/background/3.jpg</normaloff>:/background/3.jpg</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>丛林之风</string>
    </property>
    <property name="icon">
     <iconset resource="background.qrc">
      <normaloff>:/background/4.jpg</normaloff>:/background/4.jpg</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>城市之风</string>
    </property>
    <property name="icon">
     <iconset resource="background.qrc">
      <normaloff>:/background/5.jpg</normaloff>:/background/5.jpg</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>星际</string>
    </property>
    <property name="icon">
     <iconset resource="background.qrc">
      <normaloff>:/background/6.jpg</normaloff>:/background/6.jpg</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>慵懒</string>
    </property>
    <property name="icon">
     <iconset resource="background.qrc">
      <normaloff>:/background/7.jpg</normaloff>:/background/7.jpg</iconset>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="loginReturnPushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返 回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginOkPushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>修 改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginChangeButton">
   <property name="geometry">
    <rect>
     <x>448</x>
     <y>170</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>更换</string>
   </property>
  </widget>
  <widget class="QComboBox" name="loginusrLineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>65</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="loginCheckBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>142</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>loginpassLineEdit</tabstop>
  <tabstop>logincheckLineEdit</tabstop>
  <tabstop>loginloginPushButton</tabstop>
  <tabstop>loginzhucePushButton</tabstop>
  <tabstop>loginforgetPushButton</tabstop>
  <tabstop>loginexitPushbutton</tabstop>
  <tabstop>loginclosePushButton</tabstop>
 </tabstops>
 <resources>
  <include location="background.qrc"/>
 </resources>
 <connections/>
</ui>
